src/kili/cli/project/plugin.py

```diff
@@ -0,0 +1,66 @@
+"""CLI's project upload_plugin subcommand"""
+from pathlib import Path
+from typing import Dict, List, Optional, cast
+
+import click
+from typeguard import typechecked
+
+from kili.cli.common_args import Arguments, Options
+from kili.client import Kili
+from kili.exceptions import NotFound
+
+
+@click.command(name="upload_plugin")
+@Arguments.file_path
+@Options.api_key
+@Options.endpoint
+@Options.project_id
+@Options.script_name
+@Options.verbose
+@typechecked
+# pylint: disable=too-many-arguments
+def upload_plugin(
+    api_key: Optional[str],
+    endpoint: Optional[str],
+    project_id: str,
+    file_path: str,
+    script_name: Optional[str],
+    verbose: bool,
+):
+    """
+    Add a plugin to a project
+
+    file_path is the path to your file.
+
+    \b
+    !!! Examples
+        ```
+        kili project upload_plugin \\
+            main.py \\
+            --project-id <project_id>
+        ```
+    \b
+
+    """
+    kili = Kili(api_key=api_key, api_endpoint=endpoint)
```

Comment:
on the CLI, use `kili = get_kili_client(api_key=api_key, api_endpoint=endpoint)` it creates a client with the right graphQL client name (python-cli) in order to monitor from where the call come from

Reply:
done